.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: busybox-cronjob
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
```

Comment:
The concurrency policy is set to 'Allow'. This means that multiple instances of the job can run concurrently. If this is not the intended behavior, consider changing it to 'Forbid' or 'Replace'.